src/javascript/utils/compose.js

```diff
@@ -0,0 +1 @@
+export const compose = (...funcs) => funcs.reduce((a, b) => (...args) => a(b(...args)), arg => arg);
```

Comment:
I suggest to move this func to @jahia/data-helper

Reply:
Data-helper should only be composed of gql queries, react-hooks and so on... So now, It's not an option to add compose function into data-helper function.